open-metadata-distribution/open-metadata-assemblies/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>open-metadata-distribution</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>0.1-SNAPSHOT</version>
+    </parent>
+
+    <name>Open Metadata Assemblies</name>
+    <description>
+        Packaged executables from Egeria.
+    </description>
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>open-metadata-assemblies</artifactId>
+
+    <profiles>
+        <profile>
+            <id>Windows</id>
+            <activation>
+                <os>
+                    <family>windows</family>
+                </os>
+            </activation>
+            <properties>
+                <python.path.l>
+                    ${project.basedir}\src\bin;${project.basedir}\src\test\python\scripts;${project.basedir}\src\test\mock
+                </python.path.l>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>Linux</id>
+            <activation>
+                <os>
+                    <family>!windows</family>
+                </os>
+            </activation>
+            <properties>
+                <python.path.l>
+                    ${project.basedir}/src/bin:${project.basedir}/src/test/mock:${project.basedir}/src/test/python/scripts
+                </python.path.l>
+            </properties>
+        </profile>
+
+        <profile>
+            <id>dist</id>
```

Comment:
In Atlas one had to do a -Pdist to create the distribution packages. Maven has a 'package' step but in atlas this was used only for  building jars and also an 'install' step which is more about publishing the maven artifacts.

That being said if we have the right dependencies, we could remove the profile and bind to the package phase. Upside - simpler build, no need for -Pdist, always create arcives. Downloads - adds a little extra time. 

On balance I'd move to package lifecycle step & remove the profile